src/tests/ftest/cart/utest/SConscript

```diff
@@ -1,12 +1,10 @@
-#!python
 # (C) Copyright 2016-2021 Intel Corporation.
```

Comment:
Looks like the githook didn't change this copyright, or for similar SConscript files

Reply:
Looks like most SConscript files don't have this date.  I have further work on the build later so I can fix this next time.